handbook/people-ops/onboarding/git_intro.md

```diff
@@ -11,6 +11,10 @@ To visualize the git flow, there's [an article version](https://guides.github.co
 
 [Video introduction to GitHub](https://www.youtube.com/watch?v=sz6zfrQpCQg)
 
+## Unique to sourcegraph
+
+- For repositories such as `sourcegraph/sourcegraph` you should make pull requests from branches on the main repository instead of forking and making a pull request. This is due to how CI currently only runs on trusted contributors. See [external contributions](handbook/engineering/external_contributions.md)
```

Comment:
```suggestion
- For repositories such as `sourcegraph/sourcegraph` you should make pull requests from branches on the same repository instead of forking and making a pull request from your personal forks. This is because our CI currently only runs automatically for trusted contributors. See [external contributions](handbook/engineering/external_contributions.md) for more details.
```

I'm not a native speaker, feel free to tweak :)